packages/material-ui-lab/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -55,7 +55,7 @@ const InferTest = () => {
   <DatePicker<Date>
     value={null}
     onChange={(date) => {
-      expectType<Date | null, typeof date>(date);
+      expectType<Date, typeof date>(date);
```

Comment:
This should be nullable. `null` corresponds to no value.

Reply:
Should be fine now. I switched to the patch proposed in https://github.com/mui-org/material-ui/issues/26961#issuecomment-869222782.